README.md

```diff
@@ -0,0 +1,59 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Personal Potfolio
+
+> This project is about Dameh Seko as a web developer. 
+
+
+## Built With
+
+- HTML
+- CSS
+- Microverse linters
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Author
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/SekoViper/Hello-Microverse/blob/main/.gitignore.txt)
+- Twitter: [@twitterhandle](https://twitter.com/SekoViper)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/livingstone-dameh/)
```

Comment:
- [ ] Kindly customize the README file to this specific project. Please, fill the sections with instructions and information about this project and review the "Author" links and make sure they are yours and they are not broken.